src/lesson3/Lesson3Ex9.java

```diff
@@ -0,0 +1,19 @@
+package lesson3;
+
+import java.util.Scanner;
+
+public class Lesson3Ex9 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the number: ");
+
+        int num = sc.nextInt();
+        int sum = 1;
+
+        for (int i = num; i > 0; i--) {
+            sum *= i;
+        }
+            System.out.println(sum);
```

Comment:
tab-erin u coding style-in ushadir exi, save aneluc araj Alt+Ctr+L ara, to coding styled avtomat dzi inqy.